tests/src/stronk_tests.cpp

```diff
@@ -34,6 +35,146 @@ TEST(unwrap, when_unwrapping_it_works_with_its_own_argument)  // NOLINT
     EXPECT_EQ(b.unwrap<a_float_test_type>(), 42.1F);
 }
 
+template<an_int_test_type Val>
+struct type_which_requires_stronk_none_type_template_param
+{
+    constexpr static an_int_test_type value = Val;
+};
+
+TEST(none_type_template_parameter, stronk_types_can_be_used_as_none_type_template_parameters)  // NOLINT
+{
+    auto val = type_which_requires_stronk_none_type_template_param<an_int_test_type {25}> {};
+    EXPECT_EQ(val.value.unwrap<an_int_test_type>(), 25);
+}
+
+struct mark_if_moved_type
+{
+    bool* indicator;
+
+    explicit mark_if_moved_type(bool* indicator_)
+        : indicator(indicator_)
+    {
+    }
+
+    mark_if_moved_type(const mark_if_moved_type& o) = default;
+    mark_if_moved_type(mark_if_moved_type&& s) noexcept
+        : indicator(s.indicator)
+    {
+        (*this->indicator) = true;
+        s.indicator = nullptr;
+    }
+
+    auto operator=(const mark_if_moved_type&) -> mark_if_moved_type& = delete;
+    auto operator=(mark_if_moved_type&&) noexcept -> mark_if_moved_type& = delete;
+    ~mark_if_moved_type() = default;
+};
+struct a_move_indicating_type : stronk<a_move_indicating_type, mark_if_moved_type>
+{
+    using stronk::stronk;
+};
+
+TEST(move, stronk_allows_to_move_and_move_out_with_unwrap)  // NOLINT
+{
+    {  // wrapping type
+        auto marker = false;
+        auto val = a_move_indicating_type(mark_if_moved_type {&marker});
+        marker = false;
+        auto copy = val;
+        (void)copy;
```

Comment:
Is the `(void)copy;` needed? (I'm unsure which operation it would perform on the value)

Reply:
I think it's how you previously stopped linters from saying unused